### ISSUE TITLE
Form Labels translation does not allow raw content

### ISSUE THREAD

**Issue by Herod666** [symfony/symfony#35598]
**Description**  
Based on this PR: https://github.com/symfony/symfony/pull/28635
I am trying to do the following: 

```
->add('confirmedAgb', CheckboxType::class, [
                'label' => 'some.label',
                'label_translation_parameters' => [
                    '%start_link%' => '<a href="https://www.de" target="_blank">',
                    '%end_link' => '</a>'
                ],
            ])
```
but the result is 

`<a href="https://www.de" target="_blank"> link </a> here`

This function is great but it's missing this important piece.
Thanks.

**Comment by javiereguiluz** [comment]
I've missed this feature in the past too (specially in complex checkboxes/radiobuttons which contain a mult-line content to include a description + some help with HTML contents). I missed this in the `help` attribute too ... but we added `help_html` option to fix this ... so we may add a `label_html` option too.

**Comment by przemyslaw-bogusz** [comment]
I have prepared a PR for this #31375. Here is how it looks for a `label` on `CheckboxType`, depending if you set `label_html` to `false` or `true`.

![Zrzut ekranu 2020-02-9 o 17 43 05](https://user-images.githubusercontent.com/35422131/74106147-4682a000-4b64-11ea-86d6-65d7a97cf9a0.png)

It is even recognizable by Symfony plugin in Phpstorm.

![Zrzut ekranu 2020-02-9 o 17 43 34](https://user-images.githubusercontent.com/35422131/74106167-76ca3e80-4b64-11ea-920d-531a2c8cc2ff.png)

@javiereguiluz The only problem is I have a red light from Travis CI on PHP 7.4.

**Comment by ThomasLandauer** [comment]
Related issue https://github.com/symfony/symfony/issues/8733

Is it possible to rename this issue to "Add label_html option"? I was just about to open a new one ;-)

Regarding the actual feature: Whatever the arguments were for implementing `help_html` - they certainly apply to `label_html` as well :-)

**Comment by xabbuh** [comment]
Looks like we forgot to close here after merging #31375. :)

About the option name: I am not convinced that choosing something else would be a good idea given that it currently follows the same naming convention as the previously added `help_html` option.

**Comment by ThomasLandauer** [comment]
@xabbuh I wanted to rename this *issue* title, not the option ;-)

Anyway, this will be released with Symfony 5.2?

**Comment by xabbuh** [comment]
I misunderstood your comment and added the addition to the title. :)
